Core/src/org/sleuthkit/autopsy/modules/embeddedfileextractor/ImageExtractor.java

```diff
@@ -211,6 +213,13 @@ void extractImage(AbstractFile abstractFile) {
         HWPFDocument doc = null;
         try {
             doc = new HWPFDocument(new ReadContentInputStream(af));
+        } catch (OldFileFormatException ex) {
```

Comment:
Also, we have a lot of exception catch-all blocks when we are actually retrieving the data from the documents. All those catch blocks log a warning upon catching an exception. @rcordovano do we want to keep those warnings?